src/main/io/osd.c

```diff
@@ -1472,39 +1472,31 @@ static bool osdDrawSingleElement(uint8_t item)
         {
             char *p = "ACRO";
 
-            if (isAirmodeActive()) {
-                p = "AIR ";
-            }
-
-            if (FLIGHT_MODE(FAILSAFE_MODE)) {
+            if (FLIGHT_MODE(FAILSAFE_MODE))
```

Comment:
Please restore the { } coding style

Reply:
Really? The coding style standard document for INAV states the opposite. I only made this change to conform to the INAV coding style standard documentation.

Reply:
https://github.com/multiwii/baseflight/wiki/CodingStyle

> if/else blocks which only contain a single line body must not use opening/closing braces:

Reply:
Hm... I remember discussing this specific case. I'll create an issue to discuss coding style 😄

Reply:
That document is outdated and has some questionable choices that no one is following anymore. The fix is updating the document, not following it blindly.

Reply:
Sorry, was only trying to follow the documented style guide, I have no horse in this race.  I've just always followed the rule that if you touch the code, fix it as well.

Reply:
Understood, no worries :-). I just saying that Devekopment.md needs some modernization. Among other things, we should always use braces.